docs/modules/servers/pages/distributed/operate/metrics.adoc

```diff
@@ -125,9 +129,13 @@ SMTP latencies reports
 - https://github.com/apache/james-project/tree/master/server/grafana-reporting/prometheus-datasource/James_SMTP_CountBoard.json[SMTP count] :
 Request count for the SMTP protocol
 
-This is for instance how the JMAP dashboard looks like: +
-
-image::metrics.png[metrics for the JMAP protocol request latencies]
+Some example dashboards: +
```

Comment:
what is this `+` at the end of the line I see a bit everywhere?

Reply:
Enter for new line in .adoc format

Reply:
Between adoc and markdown I keep getting mixed up ahah ok :)